src/components/sidebar/components/sidebar-nav-list/index.tsx

```diff
@@ -4,8 +4,17 @@
  */
 
 import { ReactNode } from 'react'
+import classNames from 'classnames'
 import s from '../../sidebar.module.css'
 
-export default function SidebarNavList({ children }: { children: ReactNode }) {
-	return <ul className={s.navList}>{children}</ul>
+type NativeDivElement = JSX.IntrinsicElements['div']
+
+export default function SidebarNavList({
```

Comment:
Allows per-instance control of `SidebarNavList` styles. Needed for a padding adjustment for product page resources sections here:

https://github.com/hashicorp/dev-portal/pull/2568/files#diff-d30db0cc83998a5512e199bea553a8ac4f3b79ba9240e5736c491b80a32e7a44R33